locales/en-US/strings.properties

```diff
@@ -156,6 +169,14 @@ settings_pane_snippets_body=Read short and sweet updates from Mozilla about Fire
 settings_pane_done_button=Done
 settings_pane_topstories_options_sponsored=Show Sponsored Stories
 
+# LOCALIZATION NOTE(extension_notice_*): These are shown in the new Home section in about:preferences for configuring
+# the page shown on about:home and about:newtab. Each notice has a "Manage Extensions" button next to it that links to about:addons
+extension_notice_new_windows_home=An extension is controlling the page you see on new windows and home.
+extension_notice_new_tabs=An extension is controlling the page you see on new tabs.
+extension_notice_new_windows_tabs_home=An extension is controlling the page you see on new tabs, new windows, and home.
+extension_notice_multiple=There are extensions controlling the pages you see on new windows, new tabs, and home.
```

Comment:
Why "the pages"?

Reply:
This message is shown if a different extension controls about:home v.s. about:newtab. I'll add a comment to that effect

Reply:
What's the difference between this and `extension_notice_new_windows_tabs_home`, besides the fact that there are multiple extensions? I don't think it's safe to assume that one extension means one page. Also the order changes (tab-window-home, window-tab-home).

Reply:
Hm, good point. Let me check this one